Sources/AEPOptimize/Optimize+PublicAPI.swift

```diff
@@ -22,15 +22,17 @@ public extension Optimize {
     /// - Parameter decisionScopes: An array of decision scopes.
     /// - Parameter xdm: Additional XDM-formatted data to be sent in the personalization request.
     /// - Parameter data: Additional free-form data to be sent in the personalization request.
-    @objc(updatePropositions:withXdm:andData:)
-    static func updatePropositions(for decisionScopes: [DecisionScope], withXdm xdm: [String: Any]?, andData data: [String: Any]? = nil) {
+    /// - Parameter completion: Optional completion handler invoked with list of successful decision scopes and errors, if any
+    @objc(updatePropositions:withXdm:andData:completion:)
+    static func updatePropositions(for decisionScopes: [DecisionScope], withXdm xdm: [String: Any]?, andData data: [String: Any]? = nil, _ completion: (([DecisionScope]?, AEPOptimizeError?) -> Void)? = nil) {
```

Comment:
I thought we're sending back the propositions that were successfully fetched in the completion handler. Should this be 
`_ completion: (([DecisionScope: OptimizeProposition]?, AEPOptimizeError?) -> Void)? =nil`

Reply:
As discussed offline, we are now sending ([DecisionScope: OptimizeProposition]?, AEPOptimizeError?) in completion